ts/src/binance.ts

```diff
@@ -11893,6 +11682,34 @@ export default class binance extends Exchange {
         return this.safeString (ledgerType, type, type);
     }
 
+    getNetworkCodeByNetworkUrl (currencyCode: string, depositUrl: Str = undefined): Str {
```

Comment:
all coins inside their "network" entries (for each network) have some 'base-url' so we can dynamically use them to match the networks directly